File: uri/1 - Basics/1064/Main.java
import java.util.*;
class Main{
    public static void main(String []args){
        Scanner scanner = new Scanner(System.in);
        double post = 0.0;
        double soma = 0.0;
        double a = scanner.nextDouble();
        if(a >= 0.0){
             post += 1.0;
             soma += a;
        }
        double b = scanner.nextDouble();
        if(b >= 0.0){
             post += 1.0;
             soma += b;
        }
        double c = scanner.nextDouble();
        if(c >= 0.0){
             post += 1.0;
             soma += c;
        }
        double d = scanner.nextDouble();
        if(d >= 0.0){
             post += 1.0;
             soma += d;
        }
        double e = scanner.nextDouble();
        if(e >= 0.0){
             post += 1.0;
             soma += e;
        }
        double f = scanner.nextDouble();
        if(f >= 0.0){
             post += 1.0;
             soma += f;
        }
        
        System.out.printf("%.0f valores positivos\n", post);
        System.out.printf("%.1f\n", soma/post);
    }
}

File: uri/1 - Basics/1064/Main.go
package main

import (
	"fmt"
)

func main() {
	var a, b, c, d, e, f float32
	var post, soma float32

	fmt.Scanf("%f", &a)
	if a >= 0.0 {
		post += 1.0
		soma += a
	}
	fmt.Scanf("%f", &b)
	if b >= 0.0 {
		post += 1.0
		soma += b
	}
	fmt.Scanf("%f", &c)
	if c >= 0.0 {
		post += 1.0
		soma += c
	}
	fmt.Scanf("%f", &d)
	if d >= 0.0 {
		post += 1.0
		soma += d
	}
	fmt.Scanf("%f", &e)
	if e >= 0.0 {
		post += 1.0
		soma += e
	}
	fmt.Scanf("%f", &f)
	if f >= 0.0 {
		post += 1.0
		soma += f
	}
	fmt.Printf("%.0f valores positivos\n", post)
	fmt.Printf("%.1f\n", soma/post)

}
